routers/bots_api.py

```
@@ -111,6 +117,27 @@ def stream_create(request: CreateStreamRequest, response: Response):
+    response_model=CreateStreamResponse,
+    responses={
+        402: {},
+        308: {"description": "Permanent redirect to /v3/deployments/stream"},
+    },
+    operation_id=VideoBotsPage.slug_versions[0] + "__stream_create_redirect",
+    tags=["Copilot Deployments"],
+    name="Copilot Deployments Create Stream (deprecated - redirects)",
+    deprecated=True,
+    include_in_schema=True,
+)
+def stream_create_integrations(request: CreateStreamRequest, response: Response):
```

Ruff (0.14.3)

Unused function argument: `request`

(ARG001)